src/main/signal_handlers.cpp

```diff
@@ -286,43 +324,49 @@ void cvc4_init()
   act1.sa_sigaction = sigint_handler;
   act1.sa_flags = SA_SIGINFO;
   sigemptyset(&act1.sa_mask);
-  if(sigaction(SIGINT, &act1, NULL)) {
+  if (sigaction(SIGINT, &act1, NULL))
+  {
     throw Exception(string("sigaction(SIGINT) failure: ") + strerror(errno));
   }
 
   struct sigaction act2;
   act2.sa_sigaction = timeout_handler;
   act2.sa_flags = SA_SIGINFO;
   sigemptyset(&act2.sa_mask);
-  if(sigaction(SIGXCPU, &act2, NULL)) {
+  if (sigaction(SIGXCPU, &act2, NULL))
+  {
     throw Exception(string("sigaction(SIGXCPU) failure: ") + strerror(errno));
   }
 
   struct sigaction act2b;
   act2b.sa_sigaction = timeout_handler;
   act2b.sa_flags = SA_SIGINFO;
   sigemptyset(&act2b.sa_mask);
-  if(sigaction(SIGALRM, &act2b, NULL)) {
+  if (sigaction(SIGALRM, &act2b, NULL))
+  {
     throw Exception(string("sigaction(SIGALRM) failure: ") + strerror(errno));
   }
```

Comment:
I'd suggest to move the installation of SIGXCPU/SIGALRM to `install_time_limit()` to mirror the Windows side.